resources/zib/zib-TNMTumorClassification.xml

```diff
@@ -0,0 +1,617 @@
+        <map value="NL-CM:4.27.6" />
+        <comment value="TL_LymphaticInvasion" />
+      </mapping>
+    </element>
+    <element id="Observation.component:pn_PerineuralInvasion">
+      <path value="Observation.component" />
+      <sliceName value="pn_PerineuralInvasion" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:pn_PerineuralInvasion.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="371513001" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:pn_PerineuralInvasion.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="Pn_PerineuralInvasion" />
+      <definition value="Presence or absence of perineural invasion, the Pn-category. It concerns an additional description within the TNM classification that is determined by a pathologist." />
+      <alias value="Pn_PerineuraleInvasie" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.4.27.3--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-tnmtumorclassification-v1.0-2020EN" />
+        <map value="NL-CM:4.27.7" />
+        <comment value="Pn_PerineuralInvasion" />
+      </mapping>
+    </element>
+    <element id="Observation.component:v_VenousInvasion">
+      <path value="Observation.component" />
+      <sliceName value="v_VenousInvasion" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:v_VenousInvasion.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="371493002" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:v_VenousInvasion.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="V_VenousInvasion" />
+      <definition value="Presence or absence of venous invasion, the V-category. It concerns an additional description within the TNM classification that is determined by a pathologist." />
+      <alias value="V_VeneuzeInvasie" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.4.27.7--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-tnmtumorclassification-v1.0-2020EN" />
+        <map value="NL-CM:4.27.8" />
+        <comment value="V_VenousInvasion" />
+      </mapping>
+    </element>
+    <element id="Observation.component:anatomicalStage">
```

Comment:
The `.code` is identical for both this `.component` and the next one, hence it is not possible to decide whether such a `.component` belongs to the anatomical or prognostic stage. Did you already create a ticket for the ZIB center to get more specific definition codes?